flepimop/gempyor_pkg/pyproject.toml

```diff
@@ -41,6 +41,7 @@ version = "2.1"
 [project.optional-dependencies]
 aws = ["boto3", "botocore"]
 test = ["pytest", "mock"]
+docs = ["sphinx"]
```

Comment:
These lines seem out of date (since before `pylint`ing was added), could you merge in `dev` and consolidate this under the `dev` optional dependency group?